lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -59,13 +59,14 @@ def cleanup_tempfiles
       end
 
       def parse_plist(path)
-        CFPropertyList.native_types(CFPropertyList::List.new(:file => path).value)
+        CFPropertyList.native_types(CFPropertyList::List.new(file: path).value)
```

Comment:
Interesting changes in this file: did they get changed by the linter or did you make these?

Reply:
Yea that must have been linter since  I did not change helper at all.